swift_browser_ui_frontend/src/components/BrowserSupportMenu.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <c-menu
+    :items.prop="menuItems"
+    data-testid="support-menu"
+  >
+    <i
+      class="mdi mdi-help-circle-outline pr-3 menu-icon"
+    />
+    <span class="menu-active">{{ $t("message.support") }}</span>
+  </c-menu>
+</template>
+
+<script>
+import { mdiOpenInNew } from "@mdi/js"; 
+export default {
+  name: "BrowserSupportMenu",
+  data: function() {
+    return {
+      menuItems: [],
+    };
+  },
+  computed: {
+    active () {
+      return this.$store.state.active;
+    },
+    uname () {
+      return this.$store.state.uname;
+    },
+    locale () {
+      return this.$i18n.locale;
+    },
+  },
+  watch: {
+    locale () {
+      // Menu needs to be re-created & translated when locale changes
+      this.setMenu();
+    },
+  },
+  created: function () {
+    this.setMenu();
+  },
+  methods: {
+    setMenu() {
+      this.menuItems = [];
+      const rawMenuItems = [
+        {
+          label: this.$t("message.supportMenu.manual"),
+          href: "https://docs.csc.fi/data/sensitive-data/",
+        },
+        {
+          label: this.$t("message.supportMenu.billing"),
+          href: "https://research.csc.fi/pricing#buc",
+        },
+        {
+          label: this.$t("message.supportMenu.sharing"), 
+          route: {name: "TokensView", params: {        
+            user: this.uname, 
+            project: this.active.id}},
+        },
+        {
+          label: this.$t("message.supportMenu.about"), 
+          href: "https://research.csc.fi/sensitive-data",
+        }];
+      // Menu item can be hidden if it's optional rule doesn't apply
+      for (let item of rawMenuItems) {
+        this.menuItems.push({
+          name: item.label,
+          action: () => {
+            if (item.href) {
+              window.open(item.href, "_blank");
+            } else if (item.route) {
+              this.$router.push(item.route);
+            }
+          },
+          // Display external link icon for href links
+          icon: item.href && mdiOpenInNew,
+          disabled: item.disabled,
```

Comment:
Is this `disabled` property reserved for a future case ?

Reply:
Yes, in initial suggestion there was a use case for disabled item and I decided to left this implementation there. On second thought I think it would be best if there wouldn't be "extra" functionality if there's no need in current implementation.